### PR TITLE
Embed avatar

### PR DESCRIPTION
**Pull request by g3bdou**
Few changes:
Message > embed.
Author name > Avatar URL.
Added footer.

That's it, do you wish more?

### CODE REVIEW

**Comment by skeith** [comment]
My avatar cog is designed to be simple and the avatar URL needs to be clear (not masked)

**Comment by g3bdou** [comment]
It's not masked they can click on the author name and easily get the URL.

**Comment by skeith** [comment]
Here, let me rephrase. I want the URL posted as is, not as author name or anything. Plain in sight. That's what I called not masked

**Comment by g3bdou** [comment]
Well, okay 👍
